tests/test_install.py

```diff
@@ -264,6 +264,49 @@ def test_pip_args_with_windows_path(pipx_temp_env, capsys):
     assert r"D:\\TEST\\DIR" in captured.err
 
 
+@pytest.mark.parametrize("constraint_flag", ["-c ", "--constraint ", "--constraint="])
+def test_pip_args_with_constraint_relative_path(constraint_flag, pipx_temp_env, tmp_path, caplog):
+    constraint_file_name = "constraint.txt"
```

Comment:
This is the preferred name, AFAIK.

```suggestion
    constraint_file_name = "constraints.txt"
```

Reply:
haha ok, the flag on `pip` is  `constraint`, but it does not matter to me.

Changed